src/graphql/mutations/login-user.ts

```diff
@@ -1,13 +1,10 @@
-import { ApolloClient, InMemoryCache, gql } from '@apollo/client';
+import { gql } from '@apollo/client';
 
-import { uri } from '../uri';
+import { getNewClient } from '../../helpers/new-client';
 
-export const loginUser = async (email: string, password: string) => {
-  const client = new ApolloClient({
-    uri,
-    cache: new InMemoryCache(),
-  });
+const client = getNewClient({ withAuthorizarion: false });
 
+export const loginUser = async (email: string, password: string) => {
```

Comment:
```suggestion
export const loginUserMutation = async (email: string, password: string) => {
```
só pra deixar claro o que essa função faz 👍